.github/workflows/update-core-deps.yml

```diff
@@ -1,4 +1,4 @@
-name: Update core dependencies
+name: Update core dependencies and submodules
```

Comment:
The MDN submodule is also a core dependency so we can skip it. Same for the changes below.